experimental/ExteriorAlgebra/ExteriorAlgebra.jl

```diff
@@ -68,56 +100,74 @@ julia> e2*e1
 julia> (e1+e2)^2  # result is automatically reduced!
 0
 
-julia> is_zero((e1+e2)^2)
-true
-
 julia> ExtAlg, (x,y)  =  exterior_algebra(QQ, ["x","y"]);
 
 julia> y*x
 -x*y
 ```
 """
-function exterior_algebra(K::Field, NumVars::Int)
+function exterior_algebra(K::Field, numVars::Int)
   fn_name = "exterior_algebra";
-  if (NumVars < 1)  throw(ArgumentError("$fn_name ctor: (arg2) NumVars must be strictly positive"));  end;
-  return exterior_algebra(K, (1:NumVars) .|> (k -> "e$k"));
+  if (numVars < 1)  throw(ArgumentError("$fn_name ctor: (arg2) must be strictly positive, but numVars=$numVars"));  end;
```

Comment:
Idiomatic Julia would be:
```suggestion
  numVars >= 1 || throw(ArgumentError("numVars must be strictly positive, but is $numVars"))
```
In Oscar you can also use the `@req` macro:
```suggestion
  @req numVars >= 1 "numVars must be strictly positive, but is $numVars"
```
Using `fn_name` is not necessary because the stacktrace already pinpoints where the error comes from:
```
julia> exterior_algebra(QQ, 0)
ERROR: ArgumentError: exterior_algebra ctor: (arg2) NumVars must be strictly positive
Stacktrace:
 [1] exterior_algebra(K::QQField, NumVars::Int64)
   @ Oscar ~/Projekte/OSCAR/Oscar.spielwiese/experimental/ExteriorAlgebra/ExteriorAlgebra.jl:82
 [2] top-level scope
   @ REPL[15]:1
```

And if the docstring for `exterior_algebra` already mentions `NumVars` then this should be sufficient to inform the user which argument the error is about, no need to say `arg2`